custom/icds_reports/models/util.py

```diff
@@ -93,6 +93,72 @@ def run_aggregation_queries(self) -> bool:
         return False
 
 
+UCR_MAPPING = {
```

Comment:
Is there any reason to not to generate this mapping from `StaticDataSourceConfiguration` dynamically?

Reply:
Yea, at this point I guess it could be.